regrid_example.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+module load climate-utils
+alias loade3sm='source /global/common/software/e3sm/anaconda_envs/load_latest_e3sm_unified_pm-cpu.sh'
+module loade3sm
+
+# Need to run the below command twice, one for each of the SCRIP files you want to generate 
+
```

Comment:
I think we should mention that it's necessary to change the ncl script to accept the command-line arguments, e.g., 
```suggestion
# need to comment the lines 27, 31, 32 in the NCLtomakeSCRIP.ncl script so that it accepts the command-line options for the gridname, inputdir, and latlonfile variables.
```